lib/sign.js

```diff
@@ -119,6 +119,7 @@ function sign(options, config) {
       apiUrlPrefix: options.apiUrlPrefix,
       verbose: options.verbose,
       timeout: options.timeout || undefined,
+      apiProxy: options.apiProxy || undefined
```

Comment:
could you group this with all of the other `api*` prefixed options to make the list easier to scan?